modules/thermal_hydraulics/test/tests/misc/restart_1phase/test.i

```diff
@@ -101,7 +101,6 @@
     n_part_elems = 1
     materials = 'mat1'
     widths = 0.1
-    initial_T = Ts_init
```

Comment:
we could return warning on passing this during a restart

this is more in line with what you get in a regular MOOSE sim

Reply:
That's probably the right thing to do - maybe we add a parameter `override_restarted_ics` to these components that allow this behavior, `false` by default. And we could make it an error, not a warning.

Reply:
Or `allow_initial_conditions_with_restart` for consistency.

Reply:
Wait nevermind, the error message already occurs if you supply `initial_T`. We were just using `allow_initial_conditions_with_restart = true` in the test before. I'm not sure that anything needs to be done.

Reply:
but does it occur now? because you have an 'if' on isAppRestarting so I would suppose it does not occur anymore, even if we pass inital_T

Reply:
I don't know what you mean - if you supply `initial_T`, it attempts to add the IC, which triggers the error at the MOOSE level when you're restarting.